suites/reef/cephfs/tier-1_cephfs_mirror.yaml

```diff
@@ -238,3 +238,13 @@ tests:
       module: cephfs_mirroring.test_cephfs_mirror_disconnect.py
       name: Validate all failure scenarios to disconnect the mirroring
       polarion-id: "CEPH-83574100"
+  - test:
```

Comment:
Please include the test in pacific suite as well to run them in pacific and quincy releases too.